frontend/app/components/banner.tsx

```diff
@@ -7,7 +7,7 @@ interface BannerProps {
 
 export function Banner({ alert, description }: BannerProps) {
   return (
-    <div className="font-body bg-sky-800 text-white" data-testid="banner-id">
+    <div className="font-body bg-sky-950 text-white" data-testid="banner-id">
```

Comment:
The closest Tailwind color would be `bg-slate-700` based on https://find-nearest-tailwind-colour.netlify.app/. The example background color is `#26374a`.